### PR TITLE
Namespace dashboard IDs with package names

### PR DESCRIPTION
**Pull request by ycombinator**
## What does this PR do?

This PR namespaces all dashboard IDs with the package name and updates all references as well.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md#tips-for-building-integrations) and this pull request is aligned with them.
- [ ] ~I have verified that all datasets collect metrics or logs.~ Not relevant to this PR as the changes don't impact data collection.

## Authors Checklist

Check that the following packages' dashboards, **especially markdown links to other dashboards**, work as expected:

* [x] iis
* [x] netflow
* [x] nginx
* [x] system

Check that the following packages' dashboards load up in Kibana as expected, and that the package name is prefixed in the dashboard IDs in the URL:

* [x] apache
* [x] aws
* [x] cisco
* [x] iis
* [x] kafka
* [x] kubernetes
* [x] mongodb
* [x] mysql
* [x] netflow
* [x] nginx
* [x] postgresql
* [x] prometheus
* [x] rabbitmq
* [x] redis
* [x] system
* [x] windows
* [x] zookeeper

### CODE REVIEW

File: packages/system/kibana/visualization/System-Navigation-ecs.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
             "aggs": [],
             "params": {
                 "fontSize": 12,
-                "markdown": "[System Overview](#/dashboard/system-Metrics-system-overview-ecs)  | [Host Overview](#/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8-ecs) |\n[Containers overview](#/dashboard/system-CPU-slash-Memory-per-container-ecs)"
+                "markdown": "[System Overview](#/dashboard/system-Metrics-system-overview-ecs)  | [Host Overview](#/dashboard/system-system-79ffd6e0-faa0-11e6-947f-177f697178b8-ecs) |\n[Containers overview](#/dashboard/system-system-CPU-slash-Memory-per-container-ecs)"
```
Comment (ycombinator) [inline packages/system/kibana/visualization/System-Navigation-ecs.json:20]: There is duplication of `system-` here. Need to fix!

File: packages/system/kibana/visualization/fe064790-1b1f-11e7-bec4-a5e9ec5cab8b-ecs.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
                         "value": 0.85
                     }
                 ],
-                "drilldown_url": "../app/kibana#/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8-ecs?_a=(query:(language:kuery,query:'host.name:\"{{key}}\"'))",
+                "drilldown_url": "../app/kibana#/dashboard/system-system-79ffd6e0-faa0-11e6-947f-177f697178b8-ecs?_a=(query:(language:kuery,query:'host.name:\"{{key}}\"'))",
```
Comment (ycombinator) [inline packages/system/kibana/visualization/fe064790-1b1f-11e7-bec4-a5e9ec5cab8b-ecs.json:41]: There is duplication of `system-` here. Need to fix!

File: packages/system/kibana/visualization/855899e0-1b1c-11e7-b09e-037021c4f8df-ecs.json [hunk 3]
```diff
@@ -38,7 +38,7 @@
                         "value": 0.85
                     }
                 ],
-                "drilldown_url": "../app/kibana#/dashboard/system-79ffd6e0-faa0-11e6-947f-177f697178b8-ecs?_a=(query:(language:kuery,query:'host.name:\"{{key}}\"'))",
+                "drilldown_url": "../app/kibana#/dashboard/system-system-79ffd6e0-faa0-11e6-947f-177f697178b8-ecs?_a=(query:(language:kuery,query:'host.name:\"{{key}}\"'))",
```
Comment (ycombinator) [inline packages/system/kibana/visualization/855899e0-1b1c-11e7-b09e-037021c4f8df-ecs.json:41]: There is duplication of `system-` here. Need to fix!

File: packages/system/kibana/visualization/fe064790-1b1f-11e7-bec4-a5e9ec5cab8b-ecs.json (hunk 2 above)
Comment (fearful-symmetry) [reply packages/system/kibana/visualization/fe064790-1b1f-11e7-bec4-a5e9ec5cab8b-ecs.json:41]: Yah, I fixed system myself.

**Comment by ycombinator** [comment]
@mtojek @ruflin Will these changes require version bumps in the affected packages?

**Comment by andresrc** [comment]
> @mtojek @ruflin Will these changes require version bumps in the affected packages?

I think so, @ph can you confirm?

**Comment by ph** [comment]
Yes, they should be bump*

**Comment by ph** [comment]
@andresrc We could change the stability of the package beta -> experimental and bump once?

**Comment by ycombinator** [comment]
Maybe the version bump should be done as part of the PR from this repo => https://github.com/elastic/package-storage? That way all changes get lumped together in one version bump?

**Comment by ph** [comment]
Yes, just making sure we only do it once :)

**Comment by ycombinator** [comment]
Okay, I'm **not** going to bump the versions in this PR. 

Maybe we can do that in the follow up beta => experimental PR or when we promote the packages from this repo to https://github.com/elastic/package-storage.

**Comment by exekias** [comment]
Thank you for jumping into this @ycombinator! I guess we should also update the beats import script so future imports get this right?

**Comment by ycombinator** [comment]
> I guess we should also update the beats import script so future imports get this right?

Yes, and potentially also add some kind of `check` or `lint` target for checking that all dashboard IDs are prefixed with the package name.

**Pull request merged**